packaging/suse/amazon-ecs-init.spec

```diff
@@ -0,0 +1,91 @@
+#
+# spec file for package amazon-ecs-init
+#
+# Copyright (c) 2015 SUSE LINUX Products GmbH, Nuernberg, Germany.
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via http://bugs.opensuse.org/
+#
+
+%define short_name amazon-ecs
+Name:           amazon-ecs-init
+Version:        1.4.0.2
```

Comment:
Can you change this back to 1.4.0?  Our convention is to use the `Version` field to identify the version of the Agent and the `Release` field for changes to ecs-init itself or changes in the spec file.  In the case of `1.4.0-1` and `1.4.0-2`, the only difference is support for Docker 1.7.1 in the spec file.